WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabView.swift

```diff
@@ -162,6 +164,12 @@ extension ReaderTabView {
         containerView.translatesAutoresizingMaskIntoConstraints = false
         childController.view.translatesAutoresizingMaskIntoConstraints = false
 
+        controller.children.forEach {
+            $0.willMove(toParent: nil)
+            $0.view.removeFromSuperview()
+            $0.removeFromParent()
```

Comment:
I think you can use [WordPressUI's `remove()` method](https://github.com/wordpress-mobile/WordPressUI-iOS/blob/70a05f0ac69551c832e834d0d0c693936b3d27aa/WordPressUI/Extensions/UIViewController%2BChildViewController.swift#L19) here. Unless we want to shy away from introducing another dependency on that library.

```suggestion
            $0.remove()
```

Reply:
👍